ui/src/components/BallotReturn/BallotReturn.vue

```diff
@@ -47,7 +45,10 @@
                 </cv-list-item>
               </cv-list>
 
-              <span><br />Powered by the Civic Information API</span>
+              <span
```

Comment:
I don't think this one should be internationalized. It's a required string that has to be displayed when we use these results.

Reply:
Reverted back. Thank you @davidnixon